xmtp-inbox-iosTests/xmtp_inbox_iosTests.swift

```diff
@@ -8,26 +8,26 @@
 import XCTest
 
 final class xmtp_inbox_iosTests: XCTestCase {
-    override func setUpWithError() throws {
```

Comment:
These changes happened after I ran a `swiftformat .`. Do we have conflicting formatting? Or was this just missing perhaps?